backend/ipp.c

```diff
@@ -2931,9 +2931,9 @@ password_cb(const char *prompt,		/* I - Prompt (not used) */
 
 
   fprintf(stderr, "DEBUG: password_cb(prompt=\"%s\", http=%p, method=\"%s\", "
-                  "resource=\"%s\", password_tries=%p(%d)), password=%p\n",
+          "resource=\"%s\", password_tries=%p(%d)), password=%p\n",
```

Comment:
I'm not keen on losing the indentation to match up the starts of each piece of the format string. Either preserve the indentation or eliminate the newlines entirely (current coding style no longer requires manual wrapping at 80 columns...)